src/utils/work/fetchWork.js

```diff
@@ -106,7 +106,13 @@ const fetchWork = async (
   getState,
   optionals = {},
 ) => {
-  const { extras = undefined, timeout = 180, broadcast = false } = optionals;
+  const {
+    extras = undefined,
+    timeout = 180,
+    broadcast = false,
+    clusterNames = undefined,
```

Comment:
why not using these extras and adding the cluster names in them?

Reply:
FetchWork function is used for fetching every work, clusterNames are something specific to just some of the fetchWork calls, then why would we want to introduce a new parameter for fetchWork that is relevant only to some work types?

Reply:
This is exactly Agi's point which I agree with. I put the cluster names in the dot plot's work body where they are relevant. I also didn't like how I did it initially but was not sure how to pass them otherwise. I think now it should be good